src/schema/menus.schema.ts

```diff
@@ -31,4 +31,12 @@ export class MenuSchema {
   @Prop()
   @ApiProperty()
   addons: Types.ObjectId[];
+
+  @Prop({ default: null })
+  @ApiProperty()
+  created_at: Date;
```

Comment:
Should be `published_at`. We just talked about this like an hour ago wtf

Reply:
Done